Form-Controls/index.html

```diff
@@ -1,28 +1,101 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Product Pick</h1>
+    </header>
+    <main>
+      <form action="results.html" method="get" id="form-info">
+        <div class="form-control">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            required
+            placeholder="username"
+          />
+        </div>
+        <div class="form-control">
+          <label for="name">Email:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="Email"
+            required
+          />
+        </div>
 
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
-
-</html>
\ No newline at end of file
+        <div class="form-control" id="colour-radio">
+          <label>Colour:</label>
+          <div>
+            <label for="radio1" style="color: red">Red</label>
+            <input type="radio" name="colour" id="radio1" value="red" />
+          </div>
+          <div>
+            <label for="radio2" style="color: rgba(248, 244, 5, 0.907)"
```

Comment:
I really like that you added the colours here as well as the words. But when I ran Lighthouse it told me that the colour contrast did not pass its tests. How can you solve this problem?

Reply:
I found out that for background and foreground, it's important to aim for a contrast ratio of at least 4.5:1 for normal text and 3:1 for large text. I tried to adjust the colors, and while I fixed the pink and red, I had to change the yellow to brown due to the background.
![Formcontrol](https://github.com/user-attachments/assets/72273dac-c5fc-41a8-8e71-bbe99859e268)

}